library/spdm_responder_lib/libspdm_rsp_challenge_auth.c

```diff
@@ -158,8 +158,6 @@ libspdm_return_t libspdm_get_response_challenge_auth(void *context,
 
     if (slot_id == 0xFF) {
         spdm_response->header.param2 = 0;
-
-        slot_id = spdm_context->local_context.provisioned_slot_id;
```

Comment:
Since slot_id maybe 0xFF, do we need similar check above line 89?
E.g. check to ensure spdm_context->local_context.local_public_key_provision

Reply:
No need to.
The slot_id is clearly checked just at the line L81-L95 above.

Reply:
Point is taken, I will fix it by next patch set.